### ISSUE TITLE
.

### ISSUE THREAD

**Issue by Emi-TheDhamphirInLoveUnderTheFrozenStar** [brave/brave-browser#32095]

**Comment by Emi-TheDhamphirInLoveUnderTheFrozenStar** [comment]
Maybe it is the same issue as https://github.com/brave/brave-browser/issues/30854 ? That issue only talks about how rules aren't applied after launch, but maybe it is greater than that, since in this case the extension doesn't even work or anything, it is not just how rules aren't applied but how everything is broken until you disable and enable the extension.

Hope Brave team fixes that issue, it should be a priority since many extensions, seem to have the same issue.

**Comment by Emi-TheDhamphirInLoveUnderTheFrozenStar** [comment]
@rebron 

I found the build that started the issue. It was [Nightly v1.53.43 (Chromium 114.0.5735.26)](https://github.com/brave/brave-browser/releases/tag/v1.53.43) on May 15th, when Brave Nightly was upgraded to Chromium 114.
This problem is been there too long, and the more extensions support MV3, they will fail more and more.

Although the changes weren't too many from `v1.53.42` -> `v1.53.43`, there is the https://github.com/brave/brave-core/pull/17965 which upgraded to Chromium 114 and something there should break the Manifestv3 extensions using declarativeNetRequest.

I downloaded some zipped Nightly builds until I found when the extension started breaking, so I am 100% sure 1.53.43 with Chromium 114 upgrade broke it.

![image](https://github.com/brave/brave-browser/assets/122518587/8c875db6-aca3-4d9f-88e1-83ffc9c08a39)
